front/app/components/admin/ContentBuilder/Settings/index.tsx

```diff
@@ -75,8 +75,9 @@ const ContentBuilderSettings = () => {
       zIndex="99999"
       p="20px"
       w="400px"
-      h="100%"
+      h={`calc(100vh - ${stylingConsts.menuHeight}px)`}
```

Comment:
Does this mean the settings panel will only ever be this height? I think in some cases the options would cause it to be longer than the vh ? :thinking: